src/main/java/com/farion/onlinebookstore/entity/Book.java

```diff
@@ -26,6 +33,15 @@ public class Book {
     private String author;
     @Column(nullable = false, unique = true)
     private String isbn;
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Is it possible to use LAZY type?

Reply:
Yes, I was just trying to check if it works and for simplicity I put Eager, thanks